guidelines/methods/Method-font-characteristic-contrast.html

```diff
@@ -688,26 +704,28 @@ <h3>Procedure</h3>
 	<ul style="font-size: smaller">
 	<li>If an “eye dropper” type tool is used it must report values relative to the sRGB colorspace.
 	<li>The eye dropper sample should be a pixel in the middle of a major stroke of the font at the smallest size to be tested, with the content size set to default with no user scaling.
-	</ul>	
-<li>Using an automated tool like <a href="https://www.myndex.com/SAPC/">Advanced Perceptual Contrast Algorithm (APCA) Visual Contrast Demo</a> to calculate the predicted contrast between foreground text and background color.
+	</ul>
+<li>Using an automated tool like <a href="https://www.myndex.com/SAPC/">Advanced Perceptual Contrast Algorithm (APCA) Visual Contrast Demo</a> to calculate the lightness contrast between foreground text and background color.
 	<ul style="font-size: smaller">
 	<li>Important: do not swap the background or text colors in the tool entry fields.
 	<li>The APCA tool predicts contrast based in part on polarity, so it is important that the text color CSS value be entered into the text color field, and likewise for the background.	
 	</ul>	
 <li>Compare this calculated value against the lookup table "Accessible Contrast by Font Size and Weight".
-<li>Check that the absolute value of the predicted contrast percentage meets or exceeds the required value for the font weight and size.
+<li>Check that the absolute value of the lightness contrast percentage meets or exceeds the required value for the font weight and size.
 </ol>
 <h3>Accessible Contrast by Font Size and Weight</h3>
 <p>Directions:  
 <ul>
 <li>Cross index nominal font size (in CSS px) to CSS weight.  
 <li>APCA Contrast Percentage must meet or exceed the value listed. 
 <li>For light text on a dark background the APCA tool will show a negative percentage. Simply use the absolute (positive) value. For example, if the APCA value is -58%, use 58%. 	
-<li>A <span class="do-not">&osol;</span> indicates that a larger font size (or heavier font weight) must be used. 
+<li>A <span class="underWeight">&osol;</span> indicates that a larger font size (or heavier font weight) must be used. 
```

Comment:
I don't see that this CSS class is either defined or actually used anywhere else; it seems as though `do-not` is still correct (producing a red 🚫 when applied to `&osol;`). Or am I missing something?
```suggestion
<li>A <span class="do-not">&osol;</span> indicates that a larger font size (or heavier font weight) must be used. 
```

Reply:
Hmmm... the more recent CSS class is .underweight  but not super important here either way given context.